pkgs/shells/zsh/zsh-history/default.nix

```diff
@@ -29,4 +29,8 @@ buildGoModule rec {
     platforms = platforms.unix;
     maintainers = with maintainers; [ kampka ];
   };
+
+  passthru.tests = {
+    installed-tests = nixosTests.zsh-history;
```

Comment:
the installed-tests attribute is used for gnome installed tests (though it's used for freedesktop too).
Could you name it to what the actual test case is/does?

Reply:
Done